templates/stamp/utilities.sh

```diff
@@ -58,10 +61,19 @@ tune_system()
 # Configure Blob storage attached to current server 
 #############################################################################
 
+# todo: normalize tabs and spaces in this file
+
 configure_datadisks()
 {
 	# Stripe all of the data 
 	log "Formatting and configuring the data disks"
-	
+
+    # vm-disk-utils-0.1 can install mdadm which installs postfix. The postfix
+    # installation cannot be made silent using the techniques that keep the
+    # mdadm installation quiet: a) -y AND b) DEBIAN_FRONTEND=noninteractive.
+    # Therefore, we'll install postfix early with the "No configuration" option.
+    echo "postfix postfix/main_mailer_type select No configuration" | sudo debconf-set-selections
+    sudo apt-get install -y postfix
```

Comment:
yep. i added a todo above for this :) - i like making white spaces changes later in the review to cut down on noise